frontera/contrib/messagebus/zeromq/socket_config.py

```diff
@@ -1,29 +1,33 @@
 # -*- coding: utf-8 -*-
+import socket
 
 
 class SocketConfig(object):
-
-    hostname = None
-    base_port = None
+    __ip_addr = None
```

Comment:
It is not needed here, but I think it is a good convention to program as restrictive as possible. That way strange errors like name overlaps can be prohibited.
However since this is your code and you do not seem to like it, I will change it.
Still.. do you have any input on why you prefer `_` instead of `__`